packages/math-input/src/components/keypad/button.tsx

```diff
@@ -61,16 +61,21 @@ const styles = StyleSheet.create({
 export type Props = {
     onPress: () => void;
     children: React.ReactNode;
+    ariaLabel: string;
     style?: StyleType;
     tintColor?: string;
 };
 
 export default class Button extends React.Component<Props> {
     render(): React.ReactNode {
-        const {onPress, children, style, tintColor} = this.props;
+        const {onPress, ariaLabel, children, style, tintColor} = this.props;
```

Comment:
We'll need to look into this more because we don't seem to have a11y considerations accounted for, but I needed these for RTL so I figure something is better than nothing.